ssh/sandon.pub

```diff
@@ -1,2 +0,0 @@
-
-ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAwZKTrwEQ8TgYd9xTsRmUoRDpRIYe1dtRPqa3vOS3dPuZMeRbJ2KDikSEzVP2CAtBFfJBEzIlkDMon7DZ4/EhQlbQ+atNnSYWBIwfRMkwEtS4wWBAaZPNT+UszXKe7hfPqf2XhYC2z9VnlSMgi7Nbuiei9xInAIwfg5Qz5KId2H4sQ8qwVJMenXZ5SSk3jy+rsxFjBsM0B3kP+0jaDnzsG1zzd/ll1L1kCI89VzvRkayu1DbS/ErxVL8qnhmU+8JAIki28EkW+/KLT2lsIiWC2CSfJl0rd5vQJJ66I9cAo6A+21V745GvhCsuc+gwyFpkUWI7X53SrSfBmAru5WArxQ== root@sabayonx86-64
```

Comment:
@dmick this isn't a key we have, right?  don't want to inadvertantly cut off access to something that only sandon's key can get into.